src/einsteinpy/symbolic/tensor.py

```diff
@@ -66,6 +66,51 @@ def chain_config_change():
     return chain_config_change()
 
 
+def multiply(t, s, i=None, j=None):
+    """product of tensors t and s (optionally) followed by summation of i and j
```

Comment:
Docstring needs some serious changes. I will review thoroughly tomorrow Also, you will have to add tests to the PR